app/src/main/kotlin/com/wire/android/ui/debug/DebugDataOptionsViewModel.kt

```diff
@@ -45,11 +47,27 @@ import dagger.hilt.android.qualifiers.ApplicationContext
 import kotlinx.collections.immutable.toImmutableMap
 import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.launch
+import kotlinx.serialization.Serializable
 import javax.inject.Inject
 
-@Suppress("LongParameterList")
+@ViewModelScopedPreview
+interface DebugDataOptionsViewModel {
+
+    fun state(): DebugDataOptionsState = DebugDataOptionsState()
+    fun currentAccount(): UserId = UserId("value", "domain")
+    fun checkCrlRevocationList() {}
+    fun enableEncryptedProteusStorage(enabled: Boolean) {}
+    fun restartSlowSyncForRecovery() {}
+    fun enrollE2EICertificate() {}
+    fun handleE2EIEnrollmentResult(result: Either<CoreFailure, E2EIEnrollmentResult>) {}
+    fun dismissCertificateDialog() {}
+    fun forceUpdateApiVersions() {}
+    fun disableEventProcessing(disabled: Boolean) {}
+}
+
```

Comment:
the only reason to have a viewmodel interface now is to be testable, right?

Reply:
Yes, we can discuss this on collective